nxcomp/Loop.cpp

```diff
@@ -6702,12 +6761,20 @@ int WaitForRemote(int portNum)
     }
     else if (result > 0 && FD_ISSET(proxyFD, &readSet))
     {
-      sockaddr_in newAddr;
-
-      socklen_t addrLen = sizeof(sockaddr_in);
 
-      newFD = accept(proxyFD, (sockaddr *) &newAddr, &addrLen);
+      sockaddr_un newAddrUnix;
+      sockaddr_in newAddrINET;
 
+      if (socketAddress.isUnixSocket())
+      {
+        socklen_t addrLen = sizeof(sockaddr_un);
+        newFD = accept(proxyFD, (sockaddr *) &newAddrUnix, &addrLen);
```

Comment:
It seems you are not using `newAddrUnix` afterwards, so you could just pass NULL as the second `accept` argument here.